### PR TITLE
[stable/fluentd-elasticsearch] Use config from k8s repo

### PR DESCRIPTION
**Pull request by naseemkullah**
#### What this PR does / why we need it:
Logs in json format not being properly parsed in kibana.
Using config as per k8s repo fixes this. Thanks to @monotek for the info.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://developercertificate.org) signed
- [x] Chart Version bumped (if the pr is an update to an existing chart)
- [x] Variables are documented in the README.md

### CODE REVIEW

File: charts/fluentd-elasticsearch/values.yaml [hunk 1]
```diff
@@ -213,11 +213,21 @@ configMaps:
       @id fluentd-containers.log
       @type tail
       path /var/log/containers/*.log
-      pos_file /var/log/fluentd-containers.log.pos
-      time_format %Y-%m-%dT%H:%M:%S.%NZ
+      pos_file /var/log/es-containers.log.pos
```
Comment (monotek) [inline charts/fluentd-elasticsearch/values.yaml]: please don't change the pos file names.

File: charts/fluentd-elasticsearch/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: fluentd-elasticsearch
-version: 2.3.3
+version: 2.3.4
```
Comment (monotek) [inline charts/fluentd-elasticsearch/Chart.yaml]: imho we should go at least for 2.4.0 as new config is introduced.

File: charts/fluentd-elasticsearch/values.yaml [hunk 3]
```diff
@@ -249,18 +289,19 @@ configMaps:
       @type tail
       format syslog
       path /var/log/startupscript.log
-      pos_file /var/log/startupscript.log.pos
+      pos_file /var/log/es-startupscript.log.pos
```
Comment (monotek) [inline charts/fluentd-elasticsearch/values.yaml]: please don't change the pos file names.

File: charts/fluentd-elasticsearch/values.yaml [hunk 4]
```diff
@@ -249,18 +289,19 @@ configMaps:
...
     # time="2016-02-04T06:51:03.053580605Z" level=info msg="GET /containers/json"
     # time="2016-02-04T07:53:57.505612354Z" level=error msg="HTTP Error" err="No such image: -f" statusCode=404
+    # TODO(random-liu): Remove this after cri container runtime rolls out.
     <source>
       @id docker.log
       @type tail
       format /^time="(?<time>[^)]*)" level=(?<severity>[^ ]*) msg="(?<message>[^"]*)"( err="(?<error>[^"]*)")?( statusCode=($<status_code>\d+))?/
       path /var/log/docker.log
-      pos_file /var/log/docker.log.pos
+      pos_file /var/log/es-docker.log.pos
```
Comment (monotek) [inline charts/fluentd-elasticsearch/values.yaml]: please don't change the pos file names.

File: charts/fluentd-elasticsearch/values.yaml [hunk 5]
```diff
@@ -272,7 +313,7 @@ configMaps:
       # parse out of it (like severities).
       format none
       path /var/log/etcd.log
-      pos_file /var/log/etcd.log.pos
+      pos_file /var/log/es-etcd.log.pos
```
Comment (monotek) [inline charts/fluentd-elasticsearch/values.yaml]: please don't change the pos file names.

File: charts/fluentd-elasticsearch/values.yaml [hunk 6]
```diff
@@ -289,7 +330,7 @@ configMaps:
       format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
       time_format %m%d %H:%M:%S.%N
       path /var/log/kubelet.log
-      pos_file /var/log/kubelet.log.pos
+      pos_file /var/log/es-kubelet.log.pos
```
Comment (monotek) [inline charts/fluentd-elasticsearch/values.yaml]: please don't change the pos file names.

File: charts/fluentd-elasticsearch/values.yaml [hunk 7]
```diff
@@ -303,7 +344,7 @@ configMaps:
       format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
       time_format %m%d %H:%M:%S.%N
       path /var/log/kube-proxy.log
-      pos_file /var/log/kube-proxy.log.pos
+      pos_file /var/log/es-kube-proxy.log.pos
```
Comment (monotek) [inline charts/fluentd-elasticsearch/values.yaml]: please don't change the pos file names.

File: charts/fluentd-elasticsearch/values.yaml [hunk 8]
```diff
@@ -317,7 +358,7 @@ configMaps:
       format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
       time_format %m%d %H:%M:%S.%N
       path /var/log/kube-apiserver.log
-      pos_file /var/log/kube-apiserver.log.pos
+      pos_file /var/log/es-kube-apiserver.log.pos
```
Comment (monotek) [inline charts/fluentd-elasticsearch/values.yaml]: please don't change the pos file names.

File: charts/fluentd-elasticsearch/values.yaml [hunk 9]
```diff
@@ -331,7 +372,7 @@ configMaps:
       format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
       time_format %m%d %H:%M:%S.%N
       path /var/log/kube-controller-manager.log
-      pos_file /var/log/kube-controller-manager.log.pos
+      pos_file /var/log/es-kube-controller-manager.log.pos
```
Comment (monotek) [inline charts/fluentd-elasticsearch/values.yaml]: please don't change the pos file names.

File: charts/fluentd-elasticsearch/values.yaml [hunk 10]
```diff
@@ -345,24 +386,10 @@ configMaps:
       format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
       time_format %m%d %H:%M:%S.%N
       path /var/log/kube-scheduler.log
-      pos_file /var/log/kube-scheduler.log.pos
+      pos_file /var/log/es-kube-scheduler.log.pos
```
Comment (monotek) [inline charts/fluentd-elasticsearch/values.yaml]: please don't change the pos file names.

File: charts/fluentd-elasticsearch/values.yaml [hunk 11]
```diff
@@ -373,7 +400,7 @@ configMaps:
       format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
       time_format %m%d %H:%M:%S.%N
       path /var/log/glbc.log
-      pos_file /var/log/glbc.log.pos
+      pos_file /var/log/es-glbc.log.pos
```
Comment (monotek) [inline charts/fluentd-elasticsearch/values.yaml]: please don't change the pos file names.

File: charts/fluentd-elasticsearch/values.yaml [hunk 12]
```diff
@@ -387,10 +414,11 @@ configMaps:
       format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
       time_format %m%d %H:%M:%S.%N
       path /var/log/cluster-autoscaler.log
-      pos_file /var/log/cluster-autoscaler.log.pos
+      pos_file /var/log/es-cluster-autoscaler.log.pos
```
Comment (monotek) [inline charts/fluentd-elasticsearch/values.yaml]: please don't change the pos file names.

**Comment by naseemkullah** [comment]
Requested changes applied. Please review! Thanks.

**Pull request merged**